packages/mds-policy-author/request-handlers.ts

```diff
@@ -1,9 +1,9 @@
 import log from '@mds-core/mds-logger'
 import db from '@mds-core/mds-db'
-import { PolicyApiRequest, PolicyApiResponse } from './types'
 import { BadParamsError } from '@mds-core/mds-utils'
+import { PolicyApiRequest, PolicyApiResponse } from './types'
 
-const getPolicies = async (req : PolicyApiRequest, res : PolicyApiResponse) => {
+const getPolicies = async (req: PolicyApiRequest, res: PolicyApiResponse) => {
```

Comment:
should this have a type for the return value?

Reply:
Maybe, was just a linter/format change in this PR.